textile/features.textile

```diff
@@ -423,8 +429,9 @@ h3(#realtime-connection). Connection
 ** @(RTN2b)@ @echo@ should be @true@ by default; @false@ will prevent messages published by the client being echoed back
 ** @(RTN2d)@ @clientId@ contains the provided @clientId@ option of @ClientOptions@, unless @clientId@ is @null@
 ** @(RTN2e)@ Depending on the authentication scheme, either @accessToken@ contains the token string, or @key@ contains the API key
-** @(RTN2f)@ API version param @v@ should be the API version per "G4":#G4
+** @(RTN2h)@ Protocol version param @v@ must be the populated, where the value to be sent is defined in "@G5a@":#G5a
 ** @(RTN2g)@ The library and version (in the form described in "RSC7b":#RSC7b) should be included as the value of a @lib@ querystring param. For example, the 1.0.0 version of the JavaScript library would use the param @lib=js-1.0.0@
+** @(RTN2f)@ This clause has been replaced by "@RTN2h@":#RTN2h as of version 2.0 of this specification
```

Comment:
How come RTN2f has been moved from its previous position? Would it not be best for it to be alongside the point it's replaced by?

Reply:
Because of my interpretation of the 'Ordering' bullet here: https://github.com/ably/specification/blob/main/CONTRIBUTING.md#features-spec-points

I may be interpreting that wrong. Perhaps @SimonWoolf can add thoughts around this, as this convention pre-dates my time at Ably.

Reply:
Sorry, but I don't think I understand what you mean. It says "priority should be placed on ordering them in a way that makes coherent sense". For me, that would mean that a replaced spec point should be alongside the thing it's been replaced by. Please could you explain your interpretation of the spec point?